pkg/monitoring/metrics/rbd_metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	k8sv1 "k8s.io/api/core/v1"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+)
+
+var (
+	rbdMetrics = []operatormetrics.Metric{
+		vmRbdVolume,
+	}
+
+	vmRbdVolume = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: metricPrefix + "vm_rbd_volume",
+			Help: "VM with RBD mounted volume",
+			ExtraFields: map[string]string{
+				"StabilityLevel": "ALPHA",
+			},
+		},
+		[]string{"name", "namespace", "pv_name", "volume_mode", "rxbounce_enabled"},
+	)
+)
+
+func SetVmWithVolume(vm *kubevirtv1.VirtualMachine, pvc *k8sv1.PersistentVolumeClaim, pv *k8sv1.PersistentVolume) {
+	if pv.Spec.PersistentVolumeSource.CSI == nil || !strings.Contains(pv.Spec.PersistentVolumeSource.CSI.Driver, "rbd.csi.ceph.com") {
+		return
+	}
+
+	mounter, rxbounceEnabled := getAttributes(pv)
+	if mounter != "" && mounter != "rbd" {
+		return
+	}
+
+	vmRbdVolume.WithLabelValues(
+		vm.Name,
+		vm.Namespace,
+		pv.Name,
+		string(*pvc.Spec.VolumeMode),
+		strconv.FormatBool(rxbounceEnabled),
+	).Set(1)
+}
+
+func getAttributes(pv *k8sv1.PersistentVolume) (string, bool) {
```

Comment:
This function checks for 2 independent values, can you split it into 2 functions?

Reply:
removed following the next comment